airflow/dag_processing/manager.py

```diff
@@ -576,6 +578,8 @@ def _run_parsing_loop(self):
                 self.waitables.pop(sentinel)
                 self._processors.pop(processor.file_path)
 
+            if conf.getboolean("scheduler", "standalone_dag_processor"):
```

Comment:
We should read configuration option values out of the loop because they are not cached.

Reply:
Also addressed in https://github.com/apache/airflow/pull/22720